gorm-tools/src/main/groovy/gorm/tools/hibernate/QueryTimeoutConfig.groovy

```diff
@@ -0,0 +1,21 @@
+/*
+* Copyright 2024 Yak.Works - Licensed under the Apache License, Version 2.0 (the "License")
+* You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+*/
+package gorm.tools.hibernate
+
+import groovy.transform.CompileStatic
+
+import org.springframework.beans.factory.annotation.Value
+import org.springframework.boot.context.properties.ConfigurationProperties
+import org.springframework.context.annotation.Configuration
+
+@CompileStatic
+@Configuration(proxyBeanMethods = false)
+@ConfigurationProperties(prefix="yakworks.gorm.timeouts")
+class QueryTimeoutConfig {
+
+    @Value('${spring.transaction.default-timeout:-1}')
```

Comment:
add comment with how this is working. not clear. does order pick up @Value first and then set using the @ConfigurationProperties?

Reply:
Fixed, we no longer use transation timeout, and i think we shouldnt configre tht in config either.... 
as it applies to trx level, and which could override timeout at query level, preventing us from having user specific timeouts